src/components/formCovid/index.tsx

```diff
@@ -60,6 +62,16 @@ const FormCovid = (): FC => {
           id="user-access"
         />
 
+        <input
+          type="file"
+          required
+          onChange={e => {
+            setFile(e.target.files[0]);
+          }}
```

Comment:
I would not write this inline, it will become messy when having a lot of inline functions